examples/index.html

```diff
@@ -33,7 +33,6 @@
       <!-- to be migrated -->
       <li><a href="/ca-barcode-scanner"></a></li>
       <li><a href="/ca-crud"></a></li>
-      <li><a href="/ca-export"></a></li>
```

Comment:
oops, I removed that during standup to show what's left... shouldn't have been committed.